### PR TITLE
Positive, encouraging contributor expectations

### PR DESCRIPTION
**Pull request by skade**
Following todays meeting, I quickly noted down what I heard from the group.

@plevasseur and I quickly made it a round draft.

This is very much a first stab at another work document. Please liberally leave feedback.

### CODE REVIEW

File: subcommittee/coding-guidelines/contributor-expectations.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+There's two general modes of contribution that are useful to put yourself in:
+* Drafting and writing: you take over work items to work on in between meetings. You have knowledge you want to contribute.
+* Observing: you mainly look at other people producing content, and give feedback and wisdom. Maybe you're the person that observers the corner everyone else has overlooked?
+
+Its fine to switch between them given on your current constraints. It is useful to the people running the group to understand which mode you are currently in.
+
+## Some fine print
+
+The Coding Guideline group is a Rust Foundation group and follows its policies, particular its [Code of Conduct](https://foundation.rust-lang.org/policies/code-of-conduct/).
```
Comment (skade) [inline subcommittee/coding-guidelines/contributor-expectations.md:24]:
```suggestion
The Coding Guideline groups is a Rust Foundation group and follows its policies, particular its [Code of Conduct](https://foundation.rust-lang.org/policies/code-of-conduct/).
```

File: subcommittee/coding-guidelines/contributor-expectations.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+The group benefits most from:
+* People with a sense of very high quality: Our work will be consumed over multiple industries. We are a force multiplier for all good and bad things.
+* Knowledge of Rust: We only work on Rust here, so we need to assume a certain knowledge of it.
+* Knowledge of critical spaces with high impact: whether it's safety-critical or software that just impacts a lot of people - experience in high correctness and how to reach it is useful.
+* Bridge builders: You enjoy transferring the best practices between all industries.
+* Self-Organisation: There's no project managers here. You feel empowered to drive things forward that you care for.
+
+There's two general modes of contribution that are useful to put yourself in:
+* Drafting and writing: you take over work items to work on in between meetings. You have knowledge you want to contribute.
+* Observing: you mainly look at other people producing content, and give feedback and wisdom. Maybe you're the person that observers the corner everyone else has overlooked?
```
Comment (jmqd) [inline subcommittee/coding-guidelines/contributor-expectations.md:18]:
```suggestion
* Observing: you mainly look at other people producing content, and give feedback and wisdom. Maybe you're the person that observes the corner everyone else has overlooked?
```

File: subcommittee/coding-guidelines/contributor-expectations.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+Its fine to switch between them given on your current constraints. It is useful to the people running the group to understand which mode you are currently in.
```
Comment (jmqd) [inline subcommittee/coding-guidelines/contributor-expectations.md:20]:
```suggestion
It's fine to switch between them as circumstances change. It is useful to the people running the group to understand which mode you are currently in.
```

**Review by jmqd — APPROVED** [review]
LGTM, I appreciate the open framing of participating while also being clear about the goals and expectations.

Two minor typos/minor rewording comment suggestions included.

**Review by JoelMarcey — APPROVED** [review]

**Pull request merged**